packages/dev/core/src/Cameras/camera.ts

```diff
@@ -922,36 +949,78 @@ export class Camera extends Node {
                 reverseDepth
             );
         } else {
+            const computeObliqueDistance = (camera: Camera) => {
+                return (
+                    ((camera as ArcRotateCamera).radius ||

```

Comment:
I think you want to use `??` instead of `||`, so that `.radius` is used even if it's 0.

Reply:
What does it mean for a radius to be zero? I don't like using `??` because it translates to a lot of code for downlevel.

Reply:
So, the choice to use || instead of ?? was because we want a value of 0 for the radius to also evaluate to `false`.